### ISSUE TITLE
Highlight for selected elements

### ISSUE THREAD

**Issue by a19hammi** [HGustavs/LenaSYS#9847]
Elements currently being moved or edited should probably be highlighted or have its colour changed to give user feedback.

**Comment by HGustavs** [comment]
My favourite is box-markers rather than changing colors. As a color blind person, for example changing an object from red to green may be an obvious change to some people, but, conversely, red and green look like the same color to me.

<img width="378" alt="Screenshot 2021-04-06 at 14 02 12" src="https://user-images.githubusercontent.com/6978801/113707994-fdcf6400-96e0-11eb-8f1b-e8ff8a7a84cf.png">
<img width="415" alt="Screenshot 2021-04-06 at 14 02 00" src="https://user-images.githubusercontent.com/6978801/113708001-fe67fa80-96e0-11eb-9e30-1279f5775c57.png">

Example of single and multiple object selection.

ps. Ignore crappy gray shadow from my operating system.

**Comment by HGustavs** [comment]
I assume that the boxes in the selection markers above are not filled with any color. Regardless, I would like more contrast in the selection markers, for example black squares with 50% transparency to make it abundantly clear what the squares represent.

**Comment by HGustavs** [comment]
To make the big multi-object selection box, iterate over context object list and find smallest and largest X/Y and draw box around those coordinates.

**Comment by b19jenda** [comment]
In another issue different colors had been added when selecting an entity, do you want box-markers only? or box-markers and color changing?

**Comment by a19nikbo** [comment]
Fixed
